core/platform/app_identity/gae_app_identity_services.py

```diff
@@ -49,6 +49,6 @@ def get_gcs_resource_bucket_name():
         production mode.
     """
     if constants.DEV_MODE:
-        return None
+        return app_identity.get_default_gcs_bucket_name()
```

Comment:
One question -- what does this typically look like? Is it a string, or None?

Please update the docstring accordingly, too.

Reply:
It is not quite clear from the [source](https://cloud.google.com/appengine/docs/standard/python/refdocs/google.appengine.api.app_identity.app_identity). It is a string when there is a bucket available but it doesn't mention return value when bucket is absent.

Reply:
Done.

Reply:
But, in dev, is there a bucket?